src/core/__tests__/OutlookCalendar.spec.js

```diff
@@ -0,0 +1,57 @@
+import moment from 'moment'
+import { FORMAT, URL } from '../../constants'
+import CalendarBase from '../CalendarBase'
+import OutlookCalendar from '../OutlookCalendar'
+
+import queryStringToObj from '../../../test_helpers/queryStringToObj'
```

Comment:
I understand that the concerns are different, but please remove extra line breaks from `import` statements